### ISSUE TITLE
Vagrant up not working with VBOX_E_OBJECT_NOT_FOUND

### ISSUE THREAD

**Issue by naveenvalecha** [geerlingguy/drupal-vm#590]
Bringing machine 'drupalvm' up with 'virtualbox' provider...
==> drupalvm: Preparing master VM for linked clones...
    drupalvm: This is a one time operation. Once the master VM is prepared,
    drupalvm: it will be used as a base for linked clones, making the creation
    drupalvm: of new VMs take milliseconds on a modern system.
==> drupalvm: Cloning VM...
There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["clonevm", "818e8a75-0051-4440-8f74-7b4f4f7b43fc", "--register", "--name", "temp_clone_1462439444467_38077"]

Stderr: VBoxManage: error: Could not find a registered machine with UUID {818e8a75-0051-4440-8f74-7b4f4f7b43fc}
VBoxManage: error: Details: code VBOX_E_OBJECT_NOT_FOUND (0x80bb0001), component VirtualBoxWrap, interface IVirtualBox, callee nsISupports
VBoxManage: error: Context: "FindMachine(Bstr(pszSrcName).raw(), srcMachine.asOutParam())" at line 431 of file VBoxManageMisc.cpp

**Comment by oxyc** [comment]
First make sure VT-x is enabled BIOS http://docs.drupalvm.com/en/latest/other/linux/#intel-vt-x-virtualization-support. If that's not it, try https://github.com/geerlingguy/drupal-vm/issues/483#issuecomment-190219831

**Comment by geerlingguy** [comment]
@naveenvalecha - Were you ever able to confirm if this was the issue?

**Comment by naveenvalecha** [comment]
I'll test it probably by day after tomorrow.

**Comment by Badfaerie** [comment]
I had this same problem on Win 10, here's the situation and what resolved it for me:

I received the error that host updater did not have sufficient permissions and realized that, like a dork, I had failed to open babun as an administrator. 

When I did open babun as an admin and did my vagrant up, I got the above error. In the .vagrant/machines/drupalvm/virtualbox folder, everything was getting cleared out and there was nothing going on.

After a bit of Googling, I noticed that this was an issue on Macs when trying to vagrant up with sudo or as the root user. Basically, the root user didn't have the info to the .VirtualBox folder in the user's home directory. Setting the VBOX_USER_HOME path was the recommended fix and seemed to make all in the Mac world happy again.

Out of curiosity (read desperation) I set a VBOX_USER_HOME system variable to my .VirtualBox folder at C:\Users\%user%\.VirtualBox. Rebooted the system because it's Windows and that's what we do and didn't hope for much. To my shock and amazement, it actually worked this time, without any loss of blood or voodoo sacrifices.

tl:dr - This can be caused by the administrator or root user not being able to see the .VirtualBox folder. Set VBOX_USER_HOME to that path and you may resolve it.

**Comment by oxyc** [comment]
@naveenvalecha how did it turn out? Still have an issue?

**Comment by geerlingguy** [comment]
Going to close this out now; but let me know if you still can't find resolution.
